include/config/config_safeguards.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+/*****************************
+ * COMPATIBILITY SAFE GUARDS *
+ *****************************/
+
+/**
+ * NOTE: Only mess with these if you know what you're doing!
+ * These are put in place to insure that connected functionality works as intended.
+ */
+
+
+/*****************
+ * config_graphics
+ */
+
+#ifndef F3DZEX_GBI_2
```

Comment:
As mentioned before, this is incorrect. F3DEX2 also works with rej.

Reply:
oh yeah so this should be F3DEX_GBI_2?